src/main/java/org/chenliang/freepark/service/MemberService.java

```diff
@@ -27,12 +28,16 @@ public MemberResponse createMember(MemberRequest memberRequest, Tenant tenant) {
 
   public MemberResponse updateMember(Integer id, MemberRequest memberRequest, Tenant tenant) {
     Member member = memberRepository.findFirstByIdAndTenantId(id, tenant.getId())
-        .orElseThrow(() -> new ResourceNotFoundException("Member not found"));
+      .orElseThrow(() -> new ResourceNotFoundException("Member not found"));
     setMemberFields(memberRequest, member);
     member.setTenant(tenant);
     return modelMapper.map(memberRepository.save(member), MemberResponse.class);
   }
 
+  public MemberResponse updateScore(Member member) {
```

Comment:
This method is not needed, when you want to save member, you just call `memberRepository.save(member)`.
modelMapper is used only when you return specific fields to JSON response.